test/libsolidity/syntaxTests/double_variable_declaration.sol

```diff
@@ -5,4 +5,6 @@ contract test {
 	}
 }
 // ----
-// DeclarationError: (71-80): Identifier already declared.
+// Warning: (71-80): This declaration shadows an existing declaration.
+// Warning: (46-55): Unused local variable.
```

Comment:
You may want to get rid of these warnings by using ``x;`` after both declarations. Fine either way, though.